mlflow/system_metrics/metrics/cpu_monitor.py

```diff
@@ -0,0 +1,20 @@
+"""Class for monitoring CPU stats."""
+
+import psutil
+
+from mlflow.system_metrics.metrics.base_metrics_monitor import BaseMetricsMonitor
+
+
+class CPUMonitor(BaseMetricsMonitor):
+    """Class for monitoring CPU stats."""
+
+    def __init__(self, name="cpu"):
+        super().__init__(name)
+
+    def collect_metrics(self):
+        # Get CPU metrics.
+        cpu_percent = psutil.cpu_percent()
+        self._metrics["cpu_percentage"].append(cpu_percent)
+
+        system_memory = psutil.virtual_memory()._asdict()
+        self._metrics["cpu_memory_used"].append(int(system_memory["used"] / 1e6))
```

Comment:
Do we have to call this "cpu_memory"? This is a bit misleading. CPUs DO have memory (Lx cache), but we're not collecting that data here. We're keeping tracking of system memory (RAM) with the logic. 
Can we call it "system_memory"?

Reply:
`system_memory` sounds good, I copied this name from somewhere else, agree it's misleading.